### ISSUE TITLE
Merge this repo into proophessor-do

### ISSUE THREAD

**Issue by prolic** [prooph/proophessor-do-symfony#6]
Directory structure something like this:

```
- install
    - src
- zend-expressive
    - bin
    - config
    - public
    - src
    - templates
    - ...
- symfony
    - bin
    - config
    - public
    - src
    - templates
    - ...
- src
    - Model
    - ProcessManager
    - Projection
    - ...
- tests
    - Symfony
    - Expressive
    - ProophessorDo
- vendor

In the install directory we should have some composer install scripts, similar to what is at https://github.com/zendframework/zend-expressive-skeleton/

The installer asks something like:
What framework you want to use?
a) symfony
b) zend-expressive
c) both

and so on, which installs additional composer packages.

This will reduce maintenance overhead (f.e. a bugfix or feature in proophessor-do src doesn't need to get applied to two repositories) and people also have an app where they can have a direct framework comparision.

**Comment by lucascourot** [comment]
Good idea, much better than having the Domain duplicated between several repositories.

**Comment by sandrokeil** [comment]
I had a similar disucssion with @codeliner. This could resolve the problem with the model duplication.

**Comment by mablae** [comment]
I thought about this a little bit: 

Idea: Instead of having one giant repo for all frameworks, it would be the better way to rename the current `proophessor-do` to `proophessor-do-expressive` and strip out the common parts into a seperate 
package like `proophessor-do-core`. (Model, ProcessManager, Projections)

This way no dedicated installer (is anyone working on this composer plugin already?) would be needed. The specific framework implementations would just require the `prooph/proophessor-do-core` package. 

Downside would be the model been "buried" in down `/vendor/prooph/proophessor-do-core` .. 


What do you think?

**Comment by prolic** [comment]
@mablae this proposal was already under discussion and rejected by most users.

reason: the model is part of another repository and this makes adding feature much harder, because you would have to do this in at least 2 repositories.

**Comment by codeliner** [comment]
I'm working on updating the app to new symfony flex structure and new prooph bundle versions. Once that is done we can start working on this idea. I really like it.

**Comment by codeliner** [comment]
when we want to merge both repos we should remove all templating and instead work with react + redux and an API-only backend
